embed/src/main/java/orca/embed/cloudembed/controller/CloudHandler.java

```diff
@@ -797,6 +797,11 @@ public void createInterface(ComputeElement element, DomainElement edge_device, i
         ncByInterface = element.getConnectionByInterfaceName(current_intf);
         ce.setInterfaceName(ncByInterface, new_intf);
 
+        //System.out.println("SETTING NEIGHBORHOOD" );
+        //System.out.println("  Edge Device " + edge_device.getName());
+        //System.out.println("  Link Device " + link_device.getName());
+        //System.out.println("  New Intf " + new_intf.getName());
+        //System.out.println("  ncByInterface " + ncByInterface.getName());
```

Comment:
Should probably look at implementing and debug flag instead of commenting/uncommenting code blocks.